.github/workflows/build-test-macos-blockchain.yml

```diff
@@ -100,7 +100,7 @@ jobs:
     - name: Test blockchain code with pytest
       run: |
         . ./activate
-        ./venv/bin/py.test tests/blockchain/test_*.py -s -v --durations 0
+        ./venv/bin/py.test tests/blockchain/test_*.py -s -v --durations 0 -n auto
```

Comment:
these files are auto generated. I don't immediately see what you may have changed causing them to be generated differently. maybe you can rebase on top of main.

Reply:
Same here, was added by mistake. removed it.